todo_files/databaseApi.py

```diff
@@ -0,0 +1,52 @@
+import datetime
+from bson import ObjectId
+from pymongo import MongoClient
+from datetime import datetime
+import todo_files.card as Card
+
+
+class DatabaseThing:
+
+    def __init__(self, url, db):
+        self._db = MongoClient(url)[db]
+
+    def getCardData(self):
+        allCards = Card.convertListToCards(self._db['FullList'].find())
+        return allCards
+
+    def get_card_by_id(self, id):
+        card = next((card for card in self.getCardData() if str(card.id) == id), None)
```

Comment:
You could instead do `self._db['FullList'].find_one({"_id": ObjectId(id)})` which would load only that one document from MongoDB, which would be more efficient if you had lots of cards.